docs/kpod-top.1.md

```diff
@@ -0,0 +1,41 @@
+% kpod(1) kpod-top - display the running processes of a container
+% Brent Baude
+
+## NAME
+kpod top - Display the running processes of a container
+
+## SYNOPSIS
+**kpod top**
+[**--help**|**-h**]
+
+## DESCRIPTION
+Display the running process of the container. ps-OPTION can be any of the options you would pass to a Linux ps command
```

Comment:
I'm not a fan of generalities, especially if there's a difference in distro's and or changes later.   I'd really like to see us list the ones we support if at all possible.  Plus as a user I don't want to have to pull up a second man page to find the options I can use for the command.

Reply:
I know .. but it is a really huge list and I dont think we want to maintain that either.